protocol/activity_center_persistence.go

```diff
@@ -845,6 +845,37 @@ func (db sqlitePersistence) DismissActivityCenterNotifications(ids []types.HexBy
 	return updateActivityCenterState(tx, updatedAt)
 }
 
+func (db sqlitePersistence) DismissAllActivityCenterNotificationsCommunity(communityID string, updatedAt uint64) ([]*ActivityCenterNotification, error) {
```

Comment:
rename `DismissAllActivityCenterNotificationsCommunity` to `DismissAllActivityCenterNotificationsByCommunity` sounds better?